src/rules.js

```diff
@@ -0,0 +1,66 @@
+import ruleURI from 'eslint-rule-documentation'
+
+/**
+ * Stores a list of rules from ESLint
+ */
+export default class Rules {
+  /**
+   * Instantiates a Rules object, optionally with an existing list of rules
+   * @param {Array} newRules Array of Arrays of the rule and properties
+   */
+  constructor(newRules) {
+    this.rules = new Map()
+    if (newRules) {
+      this.updateRules(newRules)
```

Comment:
I'm no `Map` expert, but couldn't this be done in one step?  Something like:

```js
this.rules = new Map(newRules)
```

Reply:
Since the constructor will never be called with a parameter except in specs, sure. If this was a "real" class though we would want validation in there that whatever `newRules` is is something that actually makes sense to instantiate with.